src/catfish-tools.cpp

```diff
@@ -1,16 +1,17 @@
-#include "lcd-tools.h"
+#include "catfish-tools.h"
 
 #include <iostream>
 #include <dlfcn.h>
 #include <hybris/common/dlfcn.h>
 #include <hybris/properties/properties.h>
 #include <MGConfItem>
 
-int AsteroidOS::LCD_Tools::SyncTime() {
+namespace AsteroidOS::LCD_Tools::Catfish {
```

Comment:
It might be nicer to use: `using namespace AsteroidOS::LCD_Tools::Catfish;` here since only those functions will be used.
Similarly with the `koi` namespace.

Reply:
@beroset what do you think?

Reply:
If you look at the `koi-tools.cpp` file, closely, you'll see that not all of the functions in that file are in the namespace.  Specifically, the two `static` functions are deliberately excluded from the `AsteroidOS::LCI_Tools::Koi` namespace because they are not part of the interface.  For consistency, I made both header files the same in that regard.  Also, if we use templates later, the `using namespace` form can lead to hard-to-diagnose problems.  See https://stackoverflow.com/a/37630222/3191481